backend/tasks/tasks.router.js

```diff
@@ -1,12 +1,13 @@
 const express = require("express");
 const controller = require("./tasks.controller");
+const { validateCreateTask, validateUpdateTask  } = require("./tasks.validation");
 
 const router = express.Router();
 
 router.get("/", controller.readAll);
 router.get("/:id", controller.readById);
-router.post("/", controller.create);
-router.put("/:id", controller.updateById);
+router.post("/", validateCreateTask, controller.create);
+router.put("/:id", validateUpdateTask, controller.updateById);
 router.delete("/:id", controller.deleteById);
 
-module.exports = router;
+module.exports = router;
```

Comment:
Sempre bom ter uma linha em branco no fim do arquivo.
```suggestion
module.exports = router;

```